CODEOWNERS

```diff
@@ -257,6 +257,10 @@ extensions/filters/http/oauth2 @derekargueta @snowp
 # Thrift
 /*/extensions/filters/network/thrift_proxy @zuercher @JuniorHsu
 
+# URL Pattern Match and Rewrite
```

Comment:
Who is the best owner for this?

Reply:
as said on https://github.com/envoyproxy/envoy/pull/22459 you can put me in for now but we should see if @krajshiva can find someone from EPT to co-own